apps/user-office-backend/src/models/questionTypes/QuestionRegistry.ts

```diff
@@ -137,3 +143,20 @@ export function getDefaultAnswerValue(
 
   return definition.getDefaultAnswer(questionTemplateRelation);
 }
+
+export function getExternalApiCallData(record: DynamicRecord) {
+  const definition = getQuestionDefinition(record.data_type as DataType);
+
+  if (!definition.externalApiCall) {
+    logger.logError('Tried to make non-existing definition callback', {
+      dataType: record.data_type as DataType,
+    });
+    throw new GraphQLError('Tried to execute non-existing definition callback');
+  }
+
+  const options = definition.externalApiCall(record as DynamicRecord);
```

Comment:
I see this `externalApiCall` is an async function. Shouldn't we `await` it or I misunderstand something here.